orca-front50/src/test/groovy/com/netflix/spinnaker/orca/front50/tasks/StartPipelineTaskSpec.groovy

```diff
@@ -16,9 +16,11 @@
 
```

Comment:
@ezimanyi here are the changes for this test

Reply:
Looks good! :+1: